### PR TITLE
main folders and files setup

### PR DESCRIPTION
**Pull request by lina**

### CODE REVIEW

File: client/post/post.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div>
+
+</div>
```
Comment (codydaig) [inline client/post/post.html:3]: New Line

**Comment by AndrewKishino** [comment]
LGTM

File: client/post/post.js [hunk 2]
```diff
@@ -0,0 +1 @@
+'use strict';
```
Comment (codydaig) [inline client/post/post.js:1]: New Line

File: client/postList/postList.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<div>
+
+</div>
```
Comment (codydaig) [inline client/postList/postList.html:3]: New Line

File: client/postList/postList.js [hunk 4]
```diff
@@ -0,0 +1 @@
+'use strict';
```
Comment (codydaig) [inline client/postList/postList.js:1]: New Line

File: client/postList/searchResults.js [hunk 5]
```diff
@@ -0,0 +1 @@
+'use strict';
```
Comment (codydaig) [inline client/postList/searchResults.js:1]: New Line

File: client/searchResults/searchResults.html [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<div>
+
+</div>
```
Comment (codydaig) [inline client/searchResults/searchResults.html:3]: New Line

File: client/signin.html [hunk 7]
```diff
@@ -0,0 +1,8 @@
+<html>
+<head>
+  <title></title>
+</head>
+<body>
+
+</body>
+</html>
```
Comment (codydaig) [inline client/signin.html:8]: New Line

**Comment by codydaig** [comment]
LGTM

**Pull request closed without merging**